docs/howto/flask-async.rst

```diff
@@ -0,0 +1,54 @@
+======================================================
+How to write a Kubernetes charm for an Async Flask app
+======================================================
+
+In this how-to guide we will configure a 12-factor Flask
```

Comment:
The use of "we" suggests a tutorial, whereas in a how-to we should orient around "you" instead.